KARL_CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Delete states option in `areaOptions` in `FormPhoneNumber`.
```

Comment:
Aussi, ça mérite dans le `CHANGELOG` normal de dire qu'on a `- Fix: Fix the tiny option on Select`.